ecommerce/views.py

```diff
@@ -8,12 +8,16 @@
 from django.contrib.auth.decorators import login_required, permission_required
 from django.db import transaction
 from django.utils.translation import gettext_lazy as _
-from .models import Booking, Cart, Order, Product
+from .models import Booking, Cart, Category, Order, Product
 
-class ProductListView(generic.ListView):
-    model = Product
-    template_name = 'product_list.html'
-    paginate_by = 8
+def product_get(request):
+    product_object = Product.objects.all()
+    category_object = Category.objects.all()
```

Comment:
Chú ý đặt cho đúng số nhiều
```suggestion
    product_list = Product.objects.all()
    category_olist = Category.objects.all()
```
hoặc
```suggestion
    products = Product.objects.all()
    categories = Category.objects.all()
```